src/garage/torch/policies/base.py

```diff
@@ -16,10 +14,10 @@ class Policy(abc.ABC):
     def __init__(self, env_spec):
         self._env_spec = env_spec
 
+    @abc.abstractmethod
     def get_action(self, observation):
         """Get action given observation."""
-        return np.squeeze(
-            self.get_actions(np.expand_dims(observation, axis=0)))
+        pass
```

Comment:
By merging, do you mean to just use the np policy class instead?

Reply:
It makes much more sense to delete the np policy class and replace it with this one. We'd need to make sure that didn't break anything though.

Reply:
> We originally made this separate because there is already a `garage.np.policies.Policy`, and it's quite different from this one. For example, it uses `Parameterized` for some reason. We didn't want to break anything (or spend time refactoring `garage.np` before we knew the design of `garage.torch`), so we've left it alone for now.
> 
> I do think at some point it would make sense to have a more unified Policy API, but for now I think it's fine for them to separate. The concrete return types are different anyways.

Whoops I didn't see this comment for some reason. I'm gonna keep them separate for now then.